### PR TITLE
[RA1 Ch02]: Changing NFVI to Cloud Infrastructure

### PR DESCRIPTION
**Pull request by ASawwaf**
Changing NFVI to Cloud Infrastructure

### CODE REVIEW

File: doc/ref_arch/openstack/chapters/chapter02.md [hunk 1]
```diff
@@ -85,8 +85,8 @@ Traceability to Reference Model.
...
 | `req.inf.ntw.05` | Network | The Architecture **must** allow for East/West tenant traffic within the cloud (via tunnelled encapsulation overlay such as VXLAN or Geneve). | [RA-1 4.2.3. "Network Fabric"](./chapter04.md#423-network-fabric) |
 | `req.inf.ntw.07` | Network | The Architecture **must** support network resiliency. | [RA-1 3.4.2.2. "Network"](./chapter03.md#3422-network) |
-| `req.inf.ntw.10` | Network | The NFVI Network Fabric **must** be capable of enabling highly available (Five 9’s or better) NFVI. | [RA-1 3.4.2.2. "Network"](./chapter03.md#3422-network) |
-| `req.inf.ntw.15` | Network | The Architecture **must** support multiple networking options for NFVI to support various infrastructure profiles (Base, Network Intensive<!-- , and Compute Intensive -->).| [RA-1 4.2.3.4. "Neutron ML2-plugin Integration"](./chapter04.md#4234-neutron-ml2-integration) and ["OpenStack Neutron Plugins"](https://wiki.openstack.org/wiki/Neutron_Plugins_and_Drivers) |
+| `req.inf.ntw.10` | Network | The cloud infrastructure Network Fabric **must** be capable of enabling highly available (Five 9’s or better) cloud infrastructure. | [RA-1 3.4.2.2. "Network"](./chapter03.md#3422-network) |
```
Comment (CsatariGergely) [inline doc/ref_arch/openstack/chapters/chapter02.md]: Cloud Infrastructure is a glossary term, please indicate this by writing it came case.

**Comment by pgoyal01** [comment]
@ASawwaf One more NFVI occurrence:
[2.3 Architecture and OpenStack Requirements](https://github.com/cntt-n/CNTT/blob/ASawwaf-patch-5/doc/ref_arch/openstack/chapters/chapter02.md#23-architecture-and-openstack-requirements) the line below the heading

**Comment by rabi-abdel** [comment]
@pgoyal01 can you review/approve please

**Comment by pgoyal01** [comment]
@ASawwaf 2.3 has the sentence, ""Architecture" in this chapter refers to NFVI + VIM (as specified in Reference Model Chapter 3)." May I suggest that this be changed to:
""Architecture" in this chapter refers to Cloud infrastructure (referred to as NFVI by ETSI) + VIM (as specified in Reference Model Chapter 3)."

**Comment by ASawwaf** [comment]
> @ASawwaf 2.3 has the sentence, ""Architecture" in this chapter refers to NFVI + VIM (as specified in Reference Model Chapter 3)." May I suggest that this be changed to:
> ""Architecture" in this chapter refers to Cloud infrastructure (referred to as NFVI by ETSI) + VIM (as specified in Reference Model Chapter 3)."

@pgoyal01 , Sure , i will add it

**Comment by pgoyal01** [comment]
Fixes #1405

**Pull request merged**